fbpic/main.py

```diff
@@ -368,11 +368,6 @@ def step(self, N=1, correct_currents=True,
             for ext_field in self.external_fields:
                 ext_field.apply_expression( self.ptcl, self.time )
```

Comment:
The modifications below are a correction of a minor mistake: the particles should have their position and momenta synchronized (i.e. at the same timestep), in order to calculate the ionization.